docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version : "3.8"
+services:
+    rp:
+      container_name: rp-server
+      image: kj84park/rp-server:0.0.1
+      ports:
+        - "8080:8080"
+      restart: always
+    fido2:
+      container_name: fido2-server
+      image: kj84park/fido2-server:0.0.1
+      ports:
+        - "8081:8081"
+      restart: always
```

Comment:
In my opinion, it would be better if the images are built from the local Dockerfiles, instead of pulled from dockerhub.

Reply:
For the Dockerfiles themselves, we can use multi-stage docker builds so the build process is isolated from the local environment

Reply:
Thank you so much for the suggestion! :+1:  
Let me check.

Reply:
@ric-sapasap  I have changed the settings according to your comments. thank you.
If you have any more comments, please.

Reply:
Looks amazing. Thank you very much @kj84park!